src/inspector.rs

```diff
@@ -958,6 +977,47 @@ impl V8Inspector {
   pub fn context_destroyed(&mut self, context: Local<Context>) {
     unsafe { v8_inspector__V8Inspector__contextDestroyed(self, &*context) }
   }
+
+  #[allow(clippy::too_many_arguments)]
+  pub fn exception_thrown(
+    &mut self,
+    context: Local<Context>,
+    message: StringView,
+    exception: Local<Value>,
+    detailed_message: StringView,
+    url: StringView,
+    line_number: u32,
+    column_number: u32,
+    stack_trace: UniquePtr<V8StackTrace>,
+    script_id: i32,
+  ) -> u32 {
+    unsafe {
+      v8_inspector__V8Inspector__exceptionThrown(
+        self,
+        &*context,
+        message,
+        &*exception,
+        detailed_message,
+        url,
+        line_number,
+        column_number,
+        stack_trace.into_raw(),
+        script_id,
+      )
+    }
+  }
+
+  pub fn create_stack_trace(
+    &mut self,
+    stack_trace: Local<StackTrace>,
+  ) -> UniquePtr<V8StackTrace> {
```

Comment:
Does `create_stack_trace()` ever return nullptr?
If not, the return type should be UniqueRef.

Reply:
Yes, it does, we discussed this in https://github.com/denoland/rusty_v8/pull/717#discussion_r662570789